### PR TITLE
coreos-base/oem-vmware: add oem-cloudinit

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: coreos-base/oem-vmware/files/cloud-config.yml [hunk 1]
```diff
@@ -14,6 +14,21 @@ coreos:
...
+      runtime: yes
+      content: |
+        [Unit]
+        Description=Cloudinit from VMware Backdoor
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/coreos-cloudinit --oem=vmware
+
+    - name: systemd-networkd.service
```
Comment (marineam) [inline coreos-base/oem-vmware/files/cloud-config.yml]: What is this for?

Comment (crawford) [inline coreos-base/oem-vmware/files/cloud-config.yml]: coreos-cloudinit can write a network config.

Comment (marineam) [inline coreos-base/oem-vmware/files/cloud-config.yml]: Yes.... but cloudinit itself restarts networkd when it does so

Comment (crawford) [inline coreos-base/oem-vmware/files/cloud-config.yml]: :(

**Comment by marineam** [comment]
LGTM

**Pull request merged**